src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -148,7 +148,8 @@ public Dimension getPreferredSize() {
      *                    created.
      */
     public GroupDialog(JabRefFrame jabrefFrame, AbstractGroup editedGroup) {
-        super(jabrefFrame, Localization.lang("Edit group"), true, GroupDialog.class);
+        super(jabrefFrame, (editedGroup == null) ? Localization.lang("Add group") : Localization.lang("Edit group"),
```

Comment:
I am really no friend of the inline syntax for conditional branching. In this special case (calling super), I'll go with it though. The reason for this is it's not possible to do a proper branching before the `super` call and the line is actually quite readable.

Reply:
Thanks, i think so too. The only alternative would have been to create a separate method just to determine the name of the frame.